src/Authentication/Passwords/NothingPersonalValidator.php

```diff
@@ -72,10 +72,15 @@ protected function isNotPersonal(string $password, ?User $user): bool
             $needles = $this->strip_explode($userName);
 
             // extract local-part and domain parts from email as separate needles
-            [
-                $localPart,
-                $domain,
-            ] = explode('@', $email);
+            if (str_contains($email, '@')) {
```

Comment:
str_contains() requires PHP8. So using it here is a bug.
But we use polyfill-php80, so the test does not fail on PHP 7.4.

```
$ composer why symfony/polyfill-php80
friendsofphp/php-cs-fixer v3.16.0 requires symfony/polyfill-php80 (^1.27)
```

Reply:
But it is a transitive dependency of a dev dependency, so using it here is still a bug.